.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: "Build"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+permissions:
+  id-token: write
+  contents: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version-file: .nvmrc
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          cache: sbt
+          java-version: 11
+
+      - run: make reinstall
+
+      - name: assets test
+        run: ./tools/task-runner/runner validate test
+
+      - name: assets compile
+        run: ./tools/task-runner/runner compile
+        env:
+          NODE_ENV: production
+
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - uses: guardian/actions-riff-raff@v2
+        with:
+          projectName: dotcom:frontend
+          configPath: riff-raff.yaml
+          contentDirectories: |
+            frontend-static:
+              - static/hash
+
+      - name: Test, compile, upload to RiffRaff
+        run: |
+          java \
+          -Xmx6144M \
+          -XX:ReservedCodeCacheSize=128m \
+          -Dsbt.log.noformat=true \
+          -XX:+UseParallelGC \
+          -DAPP_SECRET="fake_secret" \
+          -Duser.timezone=Australia/Sydney \
```

Comment:
Sydney?

Reply:
This is a copy-pasta from the existing build script, [`scripts/ci`](https://github.com/guardian/frontend/blob/main/scripts/ci#L20). I think it's so we would get a warning if we write code that doesn't handle timezones well! I think it's a good idea to keep it

Reply:
Worth dropping the note in here why it exists to stop someone pulling it out at a later date?